### ISSUE TITLE
Published package doesn't work

### ISSUE THREAD

**Issue by RahulMittal2099** [ThomasBleijendaal/RapidCMS#1]
Have setup RapidCMS and made a basic CMS using it. Works great in debug mode when ran from VSCode.
Published package (release mode or debug) gives following issue:
ParentId: 
TransportConnectionId: HKzUTlV7gtQXvfU8K-bf8Q

Unhandled exception in circuit 'eO2B-NTQAi-Gj70IX3l6IVuV1EwMFgeS8cfNi1b9Jew'.

Exception: 
System.NullReferenceException: Object reference not set to an instance of an object.
   at RapidCMS.Common.Services.TreeService.GetTreeAsync(String alias, ParentPath parentPath)
   at RapidCMS.UI.Components.Shared.Tree.OnParametersSetAsync()
   at Microsoft.AspNetCore.Components.ComponentBase.CallStateHasChangedOnAsyncCompletion(Task task)
   at Microsoft.AspNetCore.Components.ComponentBase.RunInitAndSetParametersAsync()

**Comment by ThomasBleijendaal** [comment]
I have made an update available (1.4.1) which help to diagnose this issue somewhat better. There was also an issue in the JsonRepository that occurs when the application is run from a package. Do you use the JsonRepository in your CMS or did you write your own Repository?

**Comment by RahulMittal2099** [comment]
Hi Thomas
Thanks for getting back to me. The issue is solved though.

Details:
1. We wrote our own repository using EF.

2. Issue in server (Windows 2016, IIS) was due to websockets not installed.
So blazor was reverting to use SentServerEvents protocol, which failed
after attempting to do initial authentication for the connection. You might
want to look into this bug or at least ensure a better error message.

But all in all, your work on RapidCMS is a very good start. Thanks for all
your efforts on it so far.
